x-pack/plugins/security_solution/public/common/components/exceptions/viewer/exception_item/exception_details.tsx

```diff
@@ -25,15 +26,28 @@ const MyExceptionDetails = styled(EuiFlexItem)`
   ${({ theme }) => css`
     background-color: ${theme.eui.euiColorLightestShade};
     padding: ${theme.eui.euiSize};
+    .eventFiltersDescriptionList {
+      margin: ${theme.eui.euiSize} ${theme.eui.euiSize} 0 ${theme.eui.euiSize};
```

Comment:
EUI Theme template has `paddingSizes` which I think are probably more appropriate to use for margin and padding css properties. should those values be used instead here (and in other places where you defined padding/margin)?